### PR TITLE
Move gcsweb styles from inline to head

### PR DESCRIPTION
**Pull request by nebril**
Improve grid look by adding borders.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/821)

### CODE REVIEW

**Comment by nebril** [comment]
@thockin please review

**Comment by thockin** [comment]
I'm not sure this is better, but I am a web-moron.  What I really want to see is Columns auto-size so the left-most column was as wide as needed for the longest item, rather than a fixed-width.

**Comment by nebril** [comment]
This doesn't seem like a best idea, when navigating through directories, the grid width is going to change and confuse user. I added overflow-wrap for long filenames, so it doesn't flow over other columns for really long file names.

**Comment by thockin** [comment]
The flip side is that for most directories, the size and mtime are very far
away from the file name, which makes it hard to align - our eyes are no
good at traversing white-space in a straight line.

Maybe we really want alternating greybars?

**Comment by techtonik** [comment]
Screenshots would be nice. I am unable to bootstrap this on my machine. I'd add monospace font for timestamps for easy scanning.

**Comment by nebril** [comment]
Added alternating greybars, now it looks like that: 
screenshot 10

File: gcsweb/cmd/gcsweb/gcsweb.go [hunk 1]
```diff
@@ -374,9 +401,9 @@ func htmlNextButton(out io.Writer, path, marker string) error {
 
 const tmplGridHeaderText = `
 	<li class="pure-g">
-	    <div class="pure-u-1-3"><u>Name</u></div>
-   		<div class="pure-u-1-3"><u>Size</u></div>
-   		<div class="pure-u-1-3"><u>Modified</u></div>
+		<div class="pure-u-2-5 grid-head">Name</div>
+		<div class="pure-u-1-5 grid-head">Size</div>
+		<div class="pure-u-2-5 grid-head">Modified</div>
 	</li>
```
Comment (techtonik) [inline gcsweb/cmd/gcsweb/gcsweb.go]: `<li class="pure-g grid-head">` ?

**Comment by techtonik** [comment]
LGTM. At first I thought that these are HTML templates, but they are in `go`. I prefer to work with templates in separate files, but I guess it is less memory hungry to stitch them together with `go` code without loading and parsing whole template file.

**Pull request merged**